adapters/sigfox-vertx/src/main/java/org/eclipse/hono/adapter/sigfox/impl/Config.java

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.sigfox.impl;
+
+import javax.annotation.PostConstruct;
+
+import org.eclipse.hono.adapter.http.HttpProtocolAdapterProperties;
+import org.eclipse.hono.client.RequestResponseClientConfigProperties;
+import org.eclipse.hono.config.ClientConfigProperties;
+import org.eclipse.hono.service.AbstractAdapterConfig;
+import org.eclipse.hono.service.metric.MetricsTags;
+import org.springframework.beans.factory.config.ObjectFactoryCreatingFactoryBean;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.spring.autoconfigure.MeterRegistryCustomizer;
+
+/**
+ * Spring Boot configuration for the Sigfox adapter.
+ */
+@Configuration
+@EnableCaching
```

Comment:
what is this good for?

Reply:
I have no idea, copied it over from the Lora adapter.

Reply:
It seems to enable support for caching. https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-caching.html